bin/deploy

```diff
@@ -10,6 +10,17 @@ if [[ $# -ne 1 ]]; then
 	echo "Please specify an environment."
 	exit 1
 else
-	bundle exec middleman build --clean --environment="$ENV"
-	bundle exec middleman s3_sync --environment="$ENV"
+  if [[ $ENV =~ pro ]]
+  then
+    read -p "You are about to deploy to production. Are you sure? (y/n) " -n 1 -r
+    echo    # move to a new line
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+      bundle exec middleman build --clean --environment="$ENV"
+      bundle exec middleman s3_sync --environment="$ENV"
```

Comment:
Since this block promises to deploy to production, we shouldn't use the variables. In theory it would matter, but since the user has responded "yes" to the query, then we just hard code production.
```sh
bundle exec middleman build --clean --environment=production
bundle exec middleman s3_sync --environment=production
```